x-pack/test_serverless/functional/test_suites/search/config.ts

```diff
@@ -22,7 +22,6 @@ export default createTestConfig({
     `--xpack.cloud.id=ES3_FTR_TESTS:ZmFrZS1kb21haW4uY2xkLmVsc3RjLmNvJGZha2Vwcm9qZWN0aWQuZXMkZmFrZXByb2plY3RpZC5rYg==`,
     `--xpack.cloud.serverless.project_id=fakeprojectid`,
     `--xpack.cloud.serverless.project_name=ES3_FTR_TESTS`,
-    `--xpack.cloud.serverless.project_type=elasticsearch`,
     `--xpack.cloud.base_url=https://fake-cloud.elastic.co`,
     `--xpack.cloud.profile_url=/user/settings/`,
     `--xpack.cloud.billing_url=/billing/overview/`,
```

Comment:
@pheyos we probably need to move these arguments to shared config as it is relevant for all projects. But I'm not sure if we want to generate id/name of have a static value based on project type.